elixir/apps/web/lib/web/live/relay_groups/new_token.ex

```diff
@@ -197,10 +197,10 @@ defmodule Web.RelayGroups.NewToken do
 
           <div id="connection-status" class="flex justify-between items-center">
             <p class="text-sm">
-              Relay not connecting? See our <.link
-                class="text-accent-500 hover:underline"
-                href="https://www.firezone.dev/kb/administer/troubleshooting#relay-not-connecting"
-              >relay troubleshooting guide</.link>.
+              Relay not connecting? See our
+              <.website_link href="/kb">
```

Comment:
I took a quick look and saw that we don't have a section for Relay troubleshooting in our current docs, but I was wondering if we wanted to go ahead and have this link go to: `/kb/administer/troubleshooting` ?

Reply:
Yeah, it's a bit weird keeping these up to date but I fixed it